src/components/CourtOrderPoa/CourtOrderPoa.vue

```diff
@@ -1,10 +1,10 @@
 <template>
   <v-card flat class="mt-4 pa-8">
     <v-row no-gutters align="start">
-      <v-col cols="3">
+      <v-col cols="2">
```

Comment:
The other Edit UI components use 3/9 for columns. Could it the v-card margin/padding throwing things off?

Reply:
The Documents and Certify are at 2, i suppose we should fix those then.